.github/workflows/pr.yml

```diff
@@ -0,0 +1,39 @@
+name: Launch Modified Tests
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  modified:
+    runs-on: ubuntu-latest
+    outputs:
+      specs: ${{ steps.modified_files.outputs.test_paths }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+
+      - name: Launch code linter
+        run: |
+          npm install
```

Comment:
at this point separated job for running cypress is questionable.

Reply:
Maybe it should be composite action instead of a reusable workflow 🤔 https://github.blog/changelog/2021-08-25-github-actions-reduce-duplication-with-action-composition/

Reply:
I agree. I guess composite action is a little bit more complex to set up but numerous advantages compared to workflows make them worth considering, it will be easier to sell :P